apps/main/src/loan/components/PageLoanCreate/Page.tsx

```diff
@@ -54,11 +55,11 @@ const Page = () => {
   const { address } = useAccount()
   const [loaded, setLoaded] = useState(false)
 
-  const collateralDatasMapper = useStore((state) => state.collaterals.collateralDatasMapper[rChainId])
-  const { llamma, llamma: { id: llammaId = '' } = {}, displayName } = collateralDatasMapper?.[rCollateralId] ?? {}
+  const market = useMintMarket({ chainId: rChainId, marketId: rCollateralId })
+  const marketId = market?.id ?? ''
```

Comment:
The queries should all support undefined, '' is just cheating the type system

```suggestion
  const marketId = market?.id
```